tests/unit/src/kakarot/precompiles/test_sha256.py

```diff
@@ -20,6 +20,7 @@ async def sha256(starknet: Starknet):
 
 @pytest.mark.asyncio
 @pytest.mark.SHA256
+@pytest.mark.timeout(999999999)
```

Comment:
why do you need to add this?

Reply:
This specifc test was failing on my laptop due to an timeout error